ner_v2/detectors/numeral/number/standard_number_detector.py

```diff
@@ -122,13 +124,20 @@ def init_regex_and_parser(self, data_directory_path):
                 for numeral in name_variants:
                     # tuple values to corresponds to (scale, increment), for unit type, scale will always be 1.
                     self.numbers_word_map[numeral] = NumberVariant(scale=1, increment=value)
+                    # map the name of number to latin numeric value
+                    self.base_numbers_map[numeral] = value
 
             elif number_type == NUMBER_TYPE_SCALE:
                 for numeral in name_variants:
                     # tuple values to corresponds to (scale, increment), for scale type, increment will always be 0.
                     self.numbers_word_map[numeral] = NumberVariant(scale=value, increment=0)
                     # Dict map to store scale and their values
                     self.scale_map[numeral] = value
+                    # map the name of number to latin numeric value
+                    self.base_numbers_map[numeral] = value
+
+            number_text = row[NUMBER_NUMERAL_FILE_NUMBER_COLUMN_NAME]
```

Comment:
this is a common function where the numeral_constants.csv file is read and variables are populated. Since in case of Chinese we require one more variable to be populated. So adding it here as adding it in Chinese number detect will require to read same file again.